src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+
+    private int threshold;
+    private int size;
+    private Node<K,V>[] table;
+
+    private static class Node<K,V> {
+        private final int hash;
```

Comment:
for this implementation hash variable is redundant) Let's remove it

Reply:
done